src/main/webapp/script-auth.js

```diff
@@ -0,0 +1,72 @@
+// firebase configuration details (nonsecret)
+var firebaseConfig = {
+    apiKey: "AIzaSyDkYtj0ZlLE4ggunFI8c6ov_wnVuDUsY6Y",
```

Comment:
I dont think we want to check in the API key, can we put it somewhere with a gitignore file?